gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/test/java/io/gravitee/am/gateway/handler/scim/resources/users/CreateUserEndpointHandlerTest.java

```diff
@@ -264,6 +267,24 @@ public void shouldUseASelectedIdp() throws Exception {
                 "Created", null);
     }
 
+    @Test
+    public void shouldAcceptCustomGraviteeUser() throws Exception {
```

Comment:
After the `testRequest`, we should test if the `userService.create` receive a user parameter with the right additionalInformation.

Reply:
FP